src/anomalib/post_processing/visualizer.py

```diff
@@ -88,11 +88,13 @@ def visualize_batch(self, batch: dict) -> Iterator[np.ndarray]:
         Returns:
             Generator that yields a display-ready visualization for each image.
         """
-        batch_size, _num_channels, height, width = batch["image"].size()
+        batch_size = batch["image"].shape[0]
         for i in range(batch_size):
             if "image_path" in batch:
+                height, width = batch["image"].shape[-2:]
                 image = read_image(path=batch["image_path"][i], image_size=(height, width))
             elif "video_path" in batch:
+                _, height, width, _ = batch["original_image"].shape
```

Comment:
Just to follow the same convention with Lines 91 and 94, would it be an idea to do something like `height, width = batch["original_image"].shape[1:3]`

Reply:
Or, Lines 91 and 94 could possibly use Line 97's convention?